Composer/packages/client/src/components/InstallDepModal.tsx

```diff
@@ -0,0 +1,63 @@
+/* eslint-disable react/no-danger */
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/** @jsx jsx */
+import { DialogTypes, DialogWrapper } from '@bfc/ui-shared/lib/components/DialogWrapper';
+import { jsx } from '@emotion/core';
+import formatMessage from 'format-message';
+import { DefaultButton, PrimaryButton } from 'office-ui-fabric-react/lib/components/Button';
+import { DialogFooter } from 'office-ui-fabric-react/lib/components/Dialog';
+import React from 'react';
+import { Text } from 'office-ui-fabric-react/lib/Text';
+import { Link } from 'office-ui-fabric-react/lib/Link';
+import { mergeStyles } from 'office-ui-fabric-react/lib/Styling';
+
+const dialogFooterClass = mergeStyles({
+  marginTop: '25px',
+});
+
+type InstallDepModalProps = {
```

Comment:
names are confusing down below for someone else who wants to reuse this, my suggestions:
setIsOpen => onDismiss: () => void; (I assume we are never gonna tell outside that the dialog is open from inside)
text => content of type React.ReactNode so caller can render whatever
link => downloadLink
linkText => downloadLinkText
learnMore => change type to an optional learnMode: {text: string, link: string} cause they always are together and one can't be defined without other.

Reply:
isOpen doesn't seem be needed

Reply:
Updated.